src/generate/CodeModelAzImpl.ts

```diff
@@ -3433,4 +3554,238 @@ export class CodeModelCliImpl implements CodeModelAz {
     public get SDK_NoFlatten() {
         return AzConfiguration.getValue(CodeGenConstants.sdkNoFlatten);
     }
+
+    /*
+    [
+        [extension, commandGroup],
+        [commandGroup, command],
+        [command, method],
+        [method, methodParameter],
+        [method, example]
+    ]
+     */
+
+    public getModelData(
```

Comment:
Don't quite understand this function. Could you add more comments?

Reply:
I have added more comments about this function. Please check again.